lighthouse/src/main.rs

```diff
@@ -647,7 +653,7 @@ fn run<E: EthSpec>(
     };
 
     if let Some(sub_matches) = matches.subcommand_matches(account_manager::CMD) {
-        eprintln!("Running account manager for {} network", network_name);
+        info!("Running account manager for {} network", network_name);
```

Comment:
since we're moving tracing subscriber further downstream, we'll probably want to keep `eprintln`